tasks/firewall.yml

```diff
@@ -27,6 +27,5 @@
         name: fedora.linux_system_roles.firewall
       vars:
         firewall: "{{ _ha_cluster_firewall }}"
-  vars:
-    _state_value: "{{ ha_cluster_manage_firewall |
-                      ternary('enabled', 'disabled') }}"
+  when:
```

Comment:
would prefer to do this without `set_fact`:
```yaml
# SPDX-License-Identifier: MIT
---
- name: Ensure the service and the ports status with the firewall role
  include_role:
    name: fedora.linux_system_roles.firewall
  vars:
    __use_fence_fw_port: "{{ __arch == 'x86_64' and
      ('fence-virt' in ha_cluster_fence_agent_packages
       or 'fence-virt' in ha_cluster_extra_packages
       or 'fence-agents-all' in ha_cluster_fence_agent_packages
       or 'fence-agents-all' in ha_cluster_extra_packages) }}"
    __fence_fw_port: "{{ [{'port': '1229/tcp', 'state': 'enabled'}]
      if __use_fence_fw_port else [] }}"
    __ha_fw_service: [{'service': 'high-availability', 'state': 'enabled' }]
    firewall: "{{ __ha_fw_service + __fence_fw_port }}"
when:
    - ha_cluster_manage_firewall | bool
```

Reply:
I was not sure what's the best way to get the `__arch` value. So, I set it to `"{{ ansible_facts['architecture'] }}"`. Is it ok?
```
  6   vars:
  7     __arch: "{{ ansible_facts['architecture'] }}"
  8     __use_fence_fw_port: "{{ __arch == 'x86_64' and
```

Reply:
> I was not sure what's the best way to get the `__arch` value. So, I set it to `"{{ ansible_facts['architecture'] }}"`. Is it ok?
> 
> ```
>   6   vars:
>   7     __arch: "{{ ansible_facts['architecture'] }}"
>   8     __use_fence_fw_port: "{{ __arch == 'x86_64' and
> ```

Yes.